io.github.dtsudo.TuxPlanetSpeedrunAnyPercent.yml

```diff
@@ -0,0 +1,49 @@
+app-id: io.github.dtsudo.TuxPlanetSpeedrunAnyPercent
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
```

Comment:
are you sure you don't need DRI?

Reply:
Thanks; I added it now - https://github.com/flathub/flathub/pull/4148/commits/fbf7c6b9b571b436801d87ce3b5b8c2854bf5bd3